cmd/commands/root.go

```diff
@@ -84,5 +86,7 @@ func GetContext() *k8sutil.Context {
 		logging.Fatal(err)
 	}
 
+	context.Context = ctx.TODO()
```

Comment:
I thought the intention was to not use `ctx.TODO()`.

Reply:
now, using cobra root command context. Similar to multus validation in rook